src/components/CreateRecipePage.tsx

```diff
@@ -94,8 +108,13 @@ const CreateRecipePage = () => {
         />
 
         <UploadFileButton setSelectedFile={setSelectedFile} />
-
-        <Button onClick={createRecipeHandler}>Create</Button>
+        {isLoading ? (
```

Comment:
after creating user should be redirected to the recipe list page with one-time notification that recipe was created